templates/playlist/playlist-client.php

```diff
@@ -41,56 +41,61 @@
-            </form>
+       	<?php if( $mdjm_event->playlist_is_open() ) : ?>
+		<?php $event_playlist_limit = mdjm_get_playlist_limit( $mdjm_event->ID ); ?>
+		<?php $entries_in_playlist  = mdjm_count_playlist_entries( $mdjm_event->ID ); ?>
+		<?php if ( $entries_in_playlist < $event_playlist_limit || $event_playlist_limit == 0 )        :  ?>
+            		<form id="mdjm-playlist-form" name="mdjm-playlist-form" action="" method="post">
+                		<?php wp_nonce_field( 'add_playlist_entry', 'mdjm_nonce', true, true ); ?>
+                		<?php mdjm_action_field( 'add_playlist_entry' ); ?>
+                		<input type="hidden" id="entry_event" name="entry_event" value="<?php echo $mdjm_event->ID; ?>" />
+               		 	<table id="mdjm-playlist-form-table">
+                    	 	<tr>
+                        		<td class="mdjm-playlist-song-cell">
+                            			<label for="entry_song"><?php _e( 'Song', 'mobile-dj-manager' ); ?></label><br />
+                            			<input type="text" name="entry_song" id="entry_song" data-placeholder="<?php _e( 'Song Name', 'mobile-dj-manager' ); ?>" required />
+                        		</td>
+
+                        		<td class="mdjm-playlist-artist-cell">
+						<label for="entry_artist"><?php _e( 'Artist', 'mobile-dj-manager' ); ?></label><br />
+	                            		<input type="text" name="entry_artist" id="entry_artist" data-placeholder="<?php _e( 'Artist Name', 'mobile-dj-manager' ); ?>" required />
+                        		</td>
+
+	                        	<td class="mdjm-playlist-category-cell">
+        	                    		<label for="entry_category"><?php _e( 'Category', 'mobile-dj-manager' ); ?></label><br />
+                	            		<?php mdjm_playlist_category_dropdown(); ?>
+                        		</td>
+
+                       			<td class="mdjm-playlist-djnotes-cell">
+                            			<label for="mdjm_playlist_djnotes"><?php printf( __( 'Notes for your %s', 'mobile-dj-manager' ), '{artist_label}' ); ?></label><br />
+				        	<textarea name="entry_djnotes" id="entry_djnotes" data-placeholder="<?php printf( __( 'Notes for your %s', 'mobile-dj-manager' ), '{artist_label}' ); ?>"></textarea>
+	                        	</td>
+        	        	</tr>
+                		<tr>
+                    			<td class="mdjm-playlist-addnew-cell" colspan="4">
+                            			<input type="submit" name="entry_addnew" id="entry_addnew" value="<?php _e( 'Add to Playlist', 'mobile-dj-manager' ); ?>" />
+                        		</td>
+                    		</tr>
+                		</table>
+            	</form>
+		<?php else : ?>
+			<p><?php printf( __( 'You have used %s tracks our of your allowance of %s in the playlist for your %s.  Existing playlist entries are displayed below.', 'mobile-dj-manager' ), $entries_in_playlist,$event_playlist_limit, mdjm_get_label_singular( true ) ); ?></p>
```

Comment:
We need to make use of the `_n()` function here to return the correct term for 'tracks' - i.e. singular or plural.

Example;

    <?php printf(
        __( 'You have used %d %s our of your allowance of %d in the playlist for your %s.  Existing playlist entries are displayed below.', 'mobile-dj-manager' ),
        $entries_in_playlist,
        _n( 'track', 'tracks', $entries_in_playlist, 'mobile-dj-manager' ),
        $event_playlist_limit,
        mdjm_get_label_singular( true )
    ); ?>

[See here for reference](https://developer.wordpress.org/reference/functions/_n/)

Also note correct use of '%d' for numeric values versus '%s' for strings.